frontend/src/__tests__/cypress/cypress/pages/components/TableToolbar.ts

```diff
@@ -0,0 +1,19 @@
+import { Contextual } from './Contextual';
+
+export class TableToolbar extends Contextual<HTMLElement> {
+  private findToggleButton(id: string) {
+    return cy.pfSwitch(id).click();
+  }
+
+  findFilterMenuOption(id: string, name: string): Cypress.Chainable<JQuery<HTMLElement>> {
+    return this.findToggleButton(id).parents().findByRole('menuitem', { name }).click();
+  }
```

Comment:
`find` should not perform the `click` action in the page object. Leave such actions to the individual tests themselves.

It's ok to perform actions within functions that abstract out several commands. However the `find` commands should be consistent with only returned selectors.